### PR TITLE
[crashtracker] Make C example emit a file

### PR DESCRIPTION
**Pull request by danielsn**
# What does this PR do?

What it says on the tin

# Motivation

The current C example was hard to follow, since you needed an instrumented agent to see where the output went.  This is cleaner. 

# Additional Notes

Anything else we should know when reviewing?

# How to test the change?

Describe here in detail how the change can be validated.

### CODE REVIEW

**Review by morrisonlevi — APPROVED** [review]
Looks good on my side.

**Review by pawelchcki — APPROVED** [review]
